### PR TITLE
Judging view

### LINKED ISSUE #66: Judging view project Page
## Container:
- Get the data from python backend for a project
- Make api request to submit judging form

## Component:
- Display project info
- `Judging Form Component`  (not implemented in this ticket)

### PR DESCRIPTION
**Pull request by ianmah**
closes #66 

image
image

### CODE REVIEW

File: src/components/ViewProject.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+const Container = styled.div`
+  display: flex;
+`
+
+const Column = styled.div`
+  margin: 1em;
+  flex: 1;
+`
+
+const StyledYoutube = styled(Youtube)`
```
Comment (jackyzha0) [inline src/components/ViewProject.js:16]: i'm a big border radius fan :)) `border-radius: 3px` might be nice

File: src/components/Youtube.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <iframe
+      title="YouTube Video"
+      id="ytplayer"
+      type="text/html"
+      className={className}
+      src={embedUrl}
+      frameborder="0"
+      allowfullscreen
+    />
+  )
```
Comment (jackyzha0) [inline src/components/Youtube.js:17]: thoughts on stylizing this using https://developers.google.com/youtube/player_parameters#IFrame_Player_API instead, will give us more flexibility in stylization

e.g.
* modestbranding=1

File: src/components/ViewProject.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      <Column>
+        <H2>How do I judge a project?</H2>
+        <P>
+          Please judge the project on the left. Hover over the ? icons to see the scoring criteria.
+        </P>
+        <ScoreInput id="tech" label="Technology" score={score} onChange={onChange} />
+        <ScoreInput id="design" label="Design" score={score} onChange={onChange} />
+        <ScoreInput id="functionality" label="Functionality" score={score} onChange={onChange} />
+        <ScoreInput id="creativity" label="Creativity" score={score} onChange={onChange} />
+        <ScoreInput id="pitch" label="Pitch" score={score} onChange={onChange} />
```
Comment (jackyzha0) [inline src/components/ViewProject.js:77]: is there a reason this deviates from the design which uses a dropdown?

Comment (jackyzha0) [reply src/components/ViewProject.js:77]: also how hard would it be to add hover tooltips to each category for more detail on that rubric category? i feel like that might  be handy

Comment (ianmah) [reply src/components/ViewProject.js:77]: i forgot to check spec and just assumed we would use radios for this. Should I go to dropdown? I feel like radio is more intuitive for a score out of 5 but maybe thats just me

Oh yeah forgot about the tooltips

Comment (jackyzha0) [reply src/components/ViewProject.js:77]: radio is fine then, just wondering about the reasoning

File: src/components/Youtube.js (hunk 2 above)
Comment (ianmah) [reply src/components/Youtube.js:17]: Are there any benefits to this over just appending the options to the src? ie `https://youtube.com/embed/${youtubeID}?modestbranding=1`

Comment (jackyzha0) [reply src/components/Youtube.js:17]: wait didnt know you could do that lmao ok nice

**Comment by acchiang** [comment]
i noticed that the disabled buttons are still clickable, yikes mb. added a commit 564980d that fixes this, PR #146

**Review by jackyzha0 — APPROVED** [review]
pog as they say 🧞

**Pull request merged**